qemu/tests/qemu_img_bitmap.py

```diff
@@ -0,0 +1,176 @@
+import logging
```

Comment:
remove this, logging will not be used in future, use test.log instead

Reply:
Updated, thanks.